.travis.yml

```diff
@@ -7,117 +7,134 @@ env:
+          packages:
+            - gcc-5
+            - g++-5
+      env: INSTALL_TYPE='conda' DEVITO_OPENMP=0 TEST_BUNDLE='tutorials' DEVITO_ARCH=gcc-5
+    - os: linux
+      python: "3.6"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
           packages:
             - gcc-7
             - g++-7
-      env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=yask YC_CXX=g++-7 INSTALL_TYPE=conda RUN_EXAMPLES=False
-  allow_failures:
-    - os: linux
-      python: "2.7"
-      env: DEVITO_ARCH=gcc-4.9 DEVITO_OPENMP=0
-    - os: osx
-      python: "2.7"
-      env: DEVITO_ARCH=clang DEVITO_OPENMP=0
-
+      env: INSTALL_TYPE='conda' DEVITO_OPENMP=0 TEST_BUNDLE='tutorials' TEST_DOCS=1 DEVITO_ARCH=gcc-7
 addons:
   apt:
     sources:
-      - ubuntu-toolchain-r-test     # For gcc 4.9, 5 and 7
+      - ubuntu-toolchain-r-test
 
 before_install:
   # Setup anaconda
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh;
-    else
+  - if [[ ${INSTALL_TYPE} == 'conda' ]]; then
       wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+      bash miniconda.sh -b -p $HOME/miniconda;
+      export PATH="$HOME/miniconda/bin:$PATH";
+      hash -r;
+      conda config --set always_yes yes --set changeps1 no;
+      conda update -q conda;
+      conda info -a;
     fi
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
-  - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
-  # Useful for debugging any issues with conda
-  - conda info -a
 
 install:
   # Install devito with conda
-  - if [[ $INSTALL_TYPE == 'conda' ]]; then
+  - if [[ ${INSTALL_TYPE} == 'conda' ]]; then
       conda env create -q -f environment.yml python=$TRAVIS_PYTHON_VERSION;
       source activate devito;
       pip install -e .;
       conda list;
     fi
   # Install devito with pip
-  - if [[ $INSTALL_TYPE == 'pip_setup' ]]; then python setup.py install; fi
+  - if [[ ${INSTALL_TYPE} == 'pip_setup' ]]; then python setup.py install; fi
 
 before_script:
   - echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
-  - if [[ $DEVITO_BACKEND == 'yask' ]]; then
+  - if [[ ${DEVITO_BACKEND} == 'yask' ]]; then
       conda install swig; cd ../;
       git clone https://github.com/opesci/yask.git;
       cd yask; make compiler && make compiler-api; pip install -e .; cd ../devito;
     fi
 
 script:
-  - flake8 --builtins=ArgumentError .
-  # Run tests with pip
-  - if [[ $INSTALL_TYPE == 'pip_setup' ]]; then python setup.py test; fi
-  # Run test with py.tests
-  - if [[ $INSTALL_TYPE == 'conda' ]]; then py.test --cov devito tests/; fi
+  - if [[ ${TEST_FLAKE} == 1 ]]; then flake8 --builtins=ArgumentError .; fi
+  # Run tests
+  - if [[ ${TEST_BUNDLE} == 'basic_pip' ]]; then python setup.py test; fi
```

Comment:
instead of ``basic_pip`` or ``basic_conda``, why not just ``unit`` (for unit tests)? then, the if condition would have to be combined somehow with ``INSTALL_TYPE``. 
In perspective of adding performance regression tests and other stuff, this might be better

Reply:
fixed in 
cb4eafb